boards/arm/nrf9160_pca10090/nrf9160_pca10090.yaml

```diff
@@ -0,0 +1,7 @@
+identifier: nrf9160_pca10090
+name: nRF9160-PCA10090
+type: mcu
+arch: arm

```

Comment:
Should we add the RAM and FLASH properties?
`ram: 256 flash: 1024`

Reply:
We do not do this, consistently, for nRF boards. But I can do that for nrf9160_pca10090

Reply:
This is now done.